lib/normalizer.c

```diff
@@ -849,7 +849,42 @@ grn_inline static const unsigned char *
 grn_nfkc_normalize_unify_alphabet_diacritical_mark(
   const unsigned char *utf8_char, unsigned char *unified)
 {
-  // TODO: Implement function to return `unified` instead of `utf8_char`.
+  /* U+00E0 LATIN SMALL LETTER A WITH GRAVE ..
+   * U+00E5 LATIN SMALL LETTER A WITH RING ABOVE
+   *
+   * U+0101 LATIN SMALL LETTER A WITH MACRON
+   * U+0103 LATIN SMALL LETTER A WITH BREVE
+   * U+0105 LATIN SMALL LETTER A WITH OGONEK
+   *
+   * U+01CE LATIN SMALL LETTER A WITH CARON
+   * U+01DF LATIN SMALL LETTER A WITH DIAERESIS AND MACRON
+   * U+01E1 LATIN SMALL LETTER A WITH DOT ABOVE AND MACRON
+   * U+01FB LATIN SMALL LETTER A WITH RING ABOVE AND ACUTE
+   *
+   * U+0201 LATIN SMALL LETTER A WITH DOUBLE GRAVE
+   * U+0203 LATIN SMALL LETTER A WITH INVERTED BREVE
+   * U+0227 LATIN SMALL LETTER A WITH DOT ABOVE
+   *
+   * U+1E01 LATIN SMALL LETTER A WITH RING BELOW,
+   * U+1EA1 LATIN SMALL LETTER A WITH DOT BELOW ..
+   * U+1EB7 LATIN SMALL LETTER A WITH BREVE AND DOT BELOW
+   */
```

Comment:
We don't have to think of capital case letters because they have already been normalized to small letters during decomposing step before this step.

https://github.com/groonga/groonga/blob/d030603cb8c0243b2f4cb6117170eecc287bd2a9/lib/nfkc.rb#L780-L788